docs/DeveloperGuide.md

```diff
@@ -320,15 +320,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`   | User                 | View timing of upcoming events                   | I know when to go                                    |
 | `* *`   | User                 | See the most urgent events                       | I know what to do first                              |
 | `* *`   | Careless user        | Undo accidental deletion                         | I can undo deleted events                            |
-| `* *`   | Impatient person     | Search contacts                                  | I can quick find specific POC                        |
-| `* *`   | Impatient person     | Search events                                    | I can quick find specific event                      |
-| `* *`   | Lazy person          | Filter events to find similar events             | I can find similar events easily                     |
+| `* *`   | Impatient event     | Search contacts                                  | I can quick find specific POC                        |
+| `* *`   | Impatient event     | Search events                                    | I can quick find specific event                      |
+| `* *`   | Lazy event          | Filter events to find similar events             | I can find similar events easily                     |
 | `* *`   | User                 | Add all information to an event at the same time | I can save time on adding all the information        |
 | `* *`   | User                 | View events associated to each contact           | I know what events to contact the POC for            |
 | `*`      | User                 | Toggle between light and dark mode               | I can use the theme i prefer                         |
 | `*`      | User                 | Redo previous action                             | I can redo previous action                           |
 | `*`      | User                 | Use abbreviation                                 | I can quickly insert items                           |
-| `*`      | Goal oriented person | Set goals                                        | I can look at my goals                               |
+| `*`      | Goal oriented event | Set goals                                        | I can look at my goals                               |
```

Comment:
Perhaps this could be left as person instead?

Reply:
👍